lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/file/MultipartUploadStatus.json

```diff
@@ -0,0 +1,41 @@
+{
+	"description": "Describes the status of a multi-part file upload.",
+	"properties": {
+		"uploadId": {
+			"type": "string",
+			"description": "The unique identifier of a multip-part file upload."
+		},
+		"startedBy": {
+			"description": "The id of the user that started this multi-part file upload.",
+			"type": "string"
+		},
+		"startedOn": {
+			"description": "The date and time this upload was first initiated.",
+			"type": "string",
+			"format":"date-time"
+		},
+		"updatedOn": {
+			"description": "The date and time this upload was last updated.",
+			"type": "string",
+			"format":"date-time"
+		},
+		"partsState": {
+			"type": "string",
+			"description": "The concatenation of the state of each part.  Each part will have a state of 'added' (1) or 'missing' (0).  For example, given an upload composed of two parts such that the first part has been added successfully while the second part is still missing the partsState='10'."

```

Comment:
"A binary representation of the upload status of all parts comprising the file to be uploaded.  E.g. a two part file for which the first part was uploaded but the second part not would have a partsState of "10"."